js/start.md

```diff
@@ -1,4 +1,5 @@
 ---
+canonical: https://docs.amplify.aws/start
```

Comment:
Should this be https://docs.amplify.aws/start?integration=js

Reply:
Chatted with Rene to clarify this behaviour. We want customers to choose the platform/framework when they select JS on old site. Resolving.